portfolio/src/main/webapp/style.css

```diff
@@ -382,6 +382,39 @@ ul li ul li{
   top: -8%;
 }
 
+#special-features h2{
+    font-size: 3rem;
+    margin-bottom: 2rem;
+    text-align: center;
+    color: #333333;
+}
+
+#special-features h2 .text-primary{
+    color: goldenrod;
+}
+
+#special-features .button{
```

Comment:
keep the style attributes sorted.

Reply:
![CSS](https://user-images.githubusercontent.com/27080061/85787972-d713e400-b6f1-11ea-97d1-3663de19e262.JPG)